types/global-modules/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'global-modules' {
```

Comment:
There is an [`@types/global-modules`](https://www.npmjs.com/package/@types/global-modules) that looks pretty similar to this, so you should be able to just use that.